### DISCUSSION TITLE
How does the EDC align with the Data Contract Service (DCS) of GXFS

### DISCUSSION THREAD

**Discussion by drandarov-io** [eclipse-edc/.github#35] [Q&A] [answered]
Hello,

I've noticed that while the EDC states that they want to be compatible with Gaia-X Ecosystems, there is a conflict with the Data Contract Service.

Therefore, I wanted to ask how the EDC (in regards to the control plane) positions itself in relation to the ["Sovereign Data Exchange" working package of the Gaia-X Federation Services](https://www.gxfs.eu/set-of-services/#sovereign-data-exchange). E.g., will they exist as alternative solutions or will there be some kind of integration in the future.

Best regards
Dmitrij Drandarov

**Comment by mspiekermann** [comment, marked as answer]
Hi @drandarov-io! Would be great if you could elaborate on the conflict you see?

The EDC provides a framework that can be used in various solution architectures. With a focus on extensibility and possible configuration(s) it allows to serve different specifications (like the GXFS). So in terms of relation the EDC can be a foundation of implementing the GXFS specifications.

Best,
Markus

**Comment by desuti** [comment]
Have the same thoughts, guess it’s time for self hosting Gesendet mit der mobilen Mail AppAm 29.12.22 um 09:41 schrieb Viriv
                
            
            
                
                    Von: "Viriv" ***@***.***>Datum: 29. Dezember 2022An: "eclipse-edc/Connector" ***@***.***>Cc: "Yannick" ***@***.***>,"Mention" ***@***.***>Betreff: Re: [eclipse-edc/Connector] How does the EDC align with the Data Contract Service (DCS) of GXFS (Discussion eclipse-edc/Connector#2105)
                
                
I'm not even safe from nsfw spam bots on GitHub anymore?

**Reply by mspiekermann** [reply]
Obviously SPAM, so I have already flagged it and am closing the discussion due to being stale.
